<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-UDNU74E" failures="0" tests="6" name="DemoPackage.BranchesTests" time="0.011" errors="1" timestamp="2025-07-14T15:44:21 GMT+05:00" skipped="5">
  <testcase classname="DemoPackage.BranchesTests" name="addBranchLongName" time="0.000">
    <skipped/>
  </testcase> <!-- addBranchLongName -->
  <testcase classname="DemoPackage.BranchesTests" name="setUp" time="0.009">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.get(String)&quot; because &quot;this.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.get(String)" because "this.driver" is null
at DemoPackage.BranchesTests.setUp(BranchesTests.java:16)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:565)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- setUp -->
  <testcase classname="DemoPackage.BranchesTests" name="addInactiveBranch" time="0.000">
    <skipped/>
  </testcase> <!-- addInactiveBranch -->
  <testcase classname="DemoPackage.BranchesTests" name="addActiveBranch" time="0.001">
    <skipped/>
  </testcase> <!-- addActiveBranch -->
  <testcase classname="DemoPackage.BranchesTests" name="addBranchBlankName" time="0.001">
    <skipped/>
  </testcase> <!-- addBranchBlankName -->
  <testcase classname="DemoPackage.BranchesTests" name="cancelAddBranch" time="0.000">
    <skipped/>
  </testcase> <!-- cancelAddBranch -->
</testsuite> <!-- DemoPackage.BranchesTests -->
